components/ControllableExpertiseSelector.js

```diff
@@ -0,0 +1,300 @@
+/* globals $, promptError: false */
+import { useState, useEffect } from 'react'
+import IconButton from './IconButton'
+import styles from '../styles/components/ControllableExpertiseSelector.module.scss'
+import { stringToColor } from '../lib/utils'
+import useUser from '../hooks/useUser'
+
+export default function ControllableExpertiseSelector() {
+  const [keyphrases, setKeyphrases] = useState({ active: [], inactive: [] })
+  const [includedKeyphrases, setIncludedKeyphrases] = useState([])
+  const [clickedKeyphrase, setClickedKeyphrase] = useState('')
+  const [recommendedPapers, setRecommendedPapers] = useState([])
+  const [activePapers, setActivePapers] = useState({})
+  const [keyphraseColors, setKeyphraseColors] = useState({})
+  const [paperWeights, setPaperWeights] = useState({})
+  const { user } = useUser()
+
+  useEffect(() => {
+    async function fetchKeyphrases() {
+      try {
+        const keyphraseResponse = await fetch(
+          `https://devapi2.openreview.net/expertise/controllable/get_recommendations/mccallum`
+        )
```

Comment:
In this case, I think you should import `api-client.js`. This file will figure out the base url depending on the environment. The following is my guess of how it should be, but please make sure you check.
```suggestion
        const keyphraseResponse = await api.get('/expertise/controllable/get_recommendations/mccallum', {}, { accessToken })
```

Reply:
Thanks. Updated this after local testing.